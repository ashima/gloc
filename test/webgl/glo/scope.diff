--- scope.true	2011-11-17 16:41:25.000000000 -0800
+++ scope.out	2011-11-17 16:45:27.000000000 -0800
@@ -1 +1,23 @@
-error
+{
+  "glo": [ 0, 1, 0 ],
+  "target": [ "webgl", [ 1, 0, 0 ] ],
+  "meta": {
+    "author": {},
+    "license": [ "Copyright (C) 2011 . All rights reserved.", "" ],
+    "library": null,
+    "version": null,
+    "build": null
+  },
+  "units": [
+    {
+      "insym": [],
+      "outsym": [ "k", "foo" ],
+      "inmac": [],
+      "opmac": [],
+      "outmac": [],
+      "source":
+        "#line 1 GLOC_0\nprecision highp float;\n\nstruct k { float f; };\n\nint foo() { struct m { int g; }; return 1; }"
+    }
+  ],
+  "linkmap": {}
+}
