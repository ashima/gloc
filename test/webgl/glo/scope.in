precision highp float;

struct k { float f; };

int foo() { struct m { int g; }; return 1; }
