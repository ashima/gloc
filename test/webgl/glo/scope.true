{
  "glo": [ 0, 1, 0 ],
  "target": [ "webgl", [ 1, 0, 0 ] ],
  "meta": {
    "copyright": [ "", 2012, "" ],
    "author": {},
    "license": null,
    "library": null,
    "version": null,
    "build": null
  },
  "units": [
    {
      "insym": [ "x", "z" ],
      "outsym": [ "foo" ],
      "inmac": [],
      "opmac": [],
      "outmac": [],
      "source":
        "#line 1 GLOC_0\nint foo() {\n  struct m { int b; } h;\n  { struct x { int b; } e = x(3); struct q { int b; } d = q(2); h.b=d.b; }\n  x f;\n  z f;\n  return m(1).b+h.b;\n}"
    }
  ],
  "linkmap": { "0": "<stdin>#u=0" }
}
