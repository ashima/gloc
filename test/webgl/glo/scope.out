{
  "glo": [ 0, 1, 0 ],
  "target": [ "webgl", [ 1, 0, 0 ] ],
  "meta": {
    "author": {},
    "license": [ "Copyright (C) 2011 . All rights reserved.", "" ],
    "library": null,
    "version": null,
    "build": null
  },
  "units": [
    {
      "insym": [],
      "outsym": [ "k", "foo" ],
      "inmac": [],
      "opmac": [],
      "outmac": [],
      "source":
        "#line 1 GLOC_0\nprecision highp float;\n\nstruct k { float f; };\n\nint foo() { struct m { int g; }; return 1; }"
    }
  ],
  "linkmap": {}
}
